Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -1209,9 +1262,14 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
 
   auto caloPHOSCells = recoData.getPHOSCells();
   auto caloPHOSCellsTRGR = recoData.getPHOSTriggers();
-
   auto ctpDigits = recoData.getCTPDigits();
+  const auto& tinfo = pc.services().get<o2::framework::TimingInfo>();
 
+  if (mCTPReadout == 1) {
+    std::vector<o2::ctp::CTPDigit> ctpDigitsCreated;
+    createCTPReadout(recoData, ctpDigitsCreated, pc);
+    ctpDigits = std::move(ctpDigitsCreated);
```

Comment:
Not sure this will work: ctpDigits is a gsl/span while ctpDigitsCreated is a vector. Did you compile and test how it works? I would simple take the ctpDigitsCreated outside of the `if` and assign its span to `ctpDigits`